src/Space.cpp

```diff
@@ -367,7 +367,7 @@ Body *Space::FindNearestTo(const Body *b, Object::Type t) const
 {
 	Body *nearest = 0;
 	double dist = FLT_MAX;
-	for (std::list<Body *>::const_iterator i = m_bodies.begin(); i != m_bodies.end(); ++i) {
+	for (std::vector<Body *>::const_iterator i = m_bodies.begin(); i != m_bodies.end(); ++i) {
```

Comment:
Same thing here, you can replicate `const_iterator` behavior by using `for (const Body *body : m_bodies)` (incidentally, if you're using a const_iterator in a normal for-loop you want `vector.cbegin()`/`vector.cend()` instead of the normal begin/end pair.)

Reply:
done!